src/pages/Login/Login.js

```diff
@@ -1,5 +1,59 @@
 import React from 'react';
+import styled from 'styled-components';
+import { KAKAO_AUTH_URL } from './OAuth';
+import { positionTopCenter } from '../../mixin';
 
 export default function Login() {
-  return <div />;
+  return (
+    <LoginWrapper>
+      <TextBox>
+        <Title>반가워요!</Title>
+        <SubTitle>
+          지금 프렙에 가입해보세요! <br />
+          오늘부터 #프렙하자
+        </SubTitle>
+      </TextBox>
+      <KakaoButton href={KAKAO_AUTH_URL}>
+        <KakaoImage src="/images/kakao_login.png" />
+      </KakaoButton>
+    </LoginWrapper>
+  );
 }
+
+const LoginWrapper = styled.div`
+  width: 900px;
+  margin: 0 auto;
+  height: 700px;
+  margin-top: 90px;
+`;
+
+const TextBox = styled.div`
+  ${positionTopCenter}
```

Comment:
직접 상단에서 import 하지 말고 ThemeProvider를 통해 받아올 수 있도록 고쳐주세요
```suggestion
  ${({ theme }) => theme.positionTopCenter}
```